common/macresman.cpp

```diff
@@ -403,6 +485,86 @@ bool MacResManager::loadFromAppleDouble(SeekableReadStream *stream) {
 	return false;
 }
 
+bool MacResManager::getFinderInfoFromMacBinary(SeekableReadStream *stream, MacFinderInfo &outFinderInfo, MacFinderExtendedInfo &outFinderExtendedInfo) {
+
+	byte infoHeader[MBI_INFOHDR];
+
+	if (stream->read(infoHeader, MBI_INFOHDR) != MBI_INFOHDR)
+		return false;
+
+	// Check that this is actually MacBinary
```

Comment:
We already have `isMacBinary()` method, why not use that?

Reply:
Was trying to avoid having to re-read from the stream but I guess I can just move the duplicated parts to a common function.